.promu.yml

```diff
@@ -0,0 +1,15 @@
+go:
+    # Whenever the Go version is updated here, .travis.yml and
+    # .circle/config.yml should also be updated if in use.
+    version: 1.12
+repository:
+    path: github.com/prometheus/prombench
+build:
+    binaries:
+        - name: prombench
```

Comment:
I would say to simplify and make it consistent lets build all tools in the root dir
```
		- name: prombench
		  path: ./cmd/prombench
        - name: scaler
          path: ./cmd/scaler
        - name: fake-webserver
          path: ./cmd/fake-webserver
```
or if tha makes things more complicated than put all in their own folders in ./cmd/name/name

Reply:
This required some changes to how docker builds are run, I added the changes here as well opened a PR for the original Makefile.common here: https://github.com/prometheus/prometheus/pull/5669